subprojects/core/src/main/java/org/gradle/internal/html/SimpleHtmlWriter.java

```diff
@@ -52,6 +55,19 @@ public SimpleMarkupWriter startElement(String name) throws IOException {
         return super.startElement(name);
     }
 
+    /**
+     * Construct a path to another file that is relative to the directory of this HTML file.
+     *
+     * This can be used to create links to other files without requiring absolute paths to be
+     * embedded in the HTML file.
+     */
+    public String relativeLink(Path otherFile) {
```

Comment:
💭 Should the Simple HTML writer be doing the link relativization?  That doesn't seem like its responsibility, vs. just getting relativized links to write.  Not a big deal if this is easier.